src/main/java/Duke/UserInterface.java

```diff
@@ -0,0 +1,112 @@
+package Duke;
+import java.util.Scanner;
+
+public class UserInterface {
+    public String EXIT = " Bye! Hope to see you again soon :)\n";
+
+    public String LOGO = "Hello from\n ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+    public String GREETING = " Hello! I'm Duke\n What can I do for you?\n";
+    public String LINEBREAK = "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\n";
+    public String COMMAND_LIST = "list";
+    public String COMMAND_MARK = "mark";
+    public String COMMAND_UNMARK = "unmark";
+    public String COMMAND_EVENT = "event";
+    public String COMMAND_TODO = "todo";
+    public String COMMAND_DEADLINE = "deadline";
+    public String COMMAND_BYE = "bye";
+
+    private TaskManager myTaskManager;
+    private Scanner scanner;
+    private String[] splitInputs;
+
+    public UserInterface() {
+        scanner = new Scanner(System.in);
+        myTaskManager = new TaskManager();
+    }
+
+    public void runProgram() {
+        String inputs;
+        inputs = scanner.nextLine();
+        while(!(inputs.equalsIgnoreCase(COMMAND_BYE))) {
```

Comment:
It would be better to gather the command word in an explicit variable and use it instead of indexed array element.